datatable/java/datatable/src/main/java/io/cucumber/datatable/DataTableTypeRegistry.java

```diff
@@ -35,47 +37,61 @@ public BigDecimal transform(String cell) {
             }
         }));
 
-        defineDataTableType(new DataTableType(Byte.class, new TableCellTransformer<Byte>() {
+        TableCellTransformer<Byte> byteTableCellTransformer = new TableCellTransformer<Byte>() {
             @Override
             public Byte transform(String cell) {
                 return Byte.decode(cell);
             }
-        }));
+        };
+        defineDataTableType(new DataTableType(Byte.class, byteTableCellTransformer));
+        defineDataTableType(new DataTableType(byte.class, byteTableCellTransformer));
 
-        defineDataTableType(new DataTableType(Short.class, new TableCellTransformer<Short>() {
+        TableCellTransformer<Short> shortTableCellTransformer = new TableCellTransformer<Short>() {
             @Override
             public Short transform(String cell) {
                 return Short.decode(cell);
             }
-        }));
+        };
+        defineDataTableType(new DataTableType(Short.class, shortTableCellTransformer));
+        defineDataTableType(new DataTableType(short.class, shortTableCellTransformer));
 
-        defineDataTableType(new DataTableType(Integer.class, new TableCellTransformer<Integer>() {
+        TableCellTransformer<Integer> integerTableCellTransformer = new TableCellTransformer<Integer>() {
             @Override
             public Integer transform(String cell) {
                 return Integer.decode(cell);
             }
-        }));
+        };
+        defineDataTableType(new DataTableType(Integer.class, integerTableCellTransformer));
 
-        defineDataTableType(new DataTableType(Long.class, new TableCellTransformer<Long>() {
+        defineDataTableType(new DataTableType(int.class, integerTableCellTransformer));
```

Comment:
Please make sure the whitespace usage is consistent between all `defineDataTableType` calls for the same type.